src/components/select/__internal__/select-text/select-text.spec.tsx

```diff
@@ -1,7 +1,11 @@
 import { mount } from "enzyme";
 import React from "react";
 import { assertStyleMatch } from "../../../../__spec_helper__/test-utils";
-import SelectText from "./select-text.component";
+import SelectText, { SelectTextProps } from ".";
+
+function renderSelectText(props: Partial<SelectTextProps> = {}) {
```

Comment:
Is the Partial function required? As far as I can see all Props are optional anyway?

Reply:
fair point, looks like you're right - I think I needed this at some point but then must have later had to make one of the props optional which wasn't before